config.json

```diff
@@ -1414,7 +1414,38 @@
       "blurb": "TODO: add blurb for tuples concept"
     }
   ],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Batteries Included",
+      "content": "Epic & well-documented standard library. Need more? Use a PyPi package - there's one for everything.",
+      "icon": "TODO"
```

Comment:
```suggestion
      "icon": ""
```

Reply:
(This will try to render the `TODO` icon if left like this, so better to be `""`)